packages/react-components/react-teaching-popover/library/src/components/TeachingPopoverCarousel/Carousel/Carousel.tsx

```diff
@@ -1,33 +1,19 @@
 import * as React from 'react';
-import {
-  isHTMLElement,
-  useMergedRefs,
-  useControllableState,
-  type EventHandler,
-  useEventCallback,
-} from '@fluentui/react-utilities';
-import { useFluent_unstable as useFluent } from '@fluentui/react-shared-contexts';
+import { isHTMLElement, useMergedRefs, useControllableState, useEventCallback } from '@fluentui/react-utilities';
+import { useAnnounce, useFluent_unstable as useFluent } from '@fluentui/react-shared-contexts';
 
 import { CAROUSEL_ITEM } from './constants';
 import { useCarouselWalker_unstable } from './useCarouselWalker';
 import { createCarouselStore } from './createCarouselStore';
-import type { CarouselValueChangeData } from './Carousel.types';
+import type { UseCarouselOptions } from './Carousel.types';
 import { CarouselContextValue } from './CarouselContext';
 
-export type UseCarouselOptions = {
-  defaultValue?: string;
-  value?: string;
-
-  onValueChange?: EventHandler<CarouselValueChangeData>;
-  onFinish?: EventHandler<CarouselValueChangeData>;
-};
```

Comment:
I wrangled types a little -- moved these to the types file and referenced them from TeachingPopoverCarousel types to reduce duplication, but let me know if you want to keep this as-is and just add `announcement` both here and to the TeachingPopoverCarousel types

Reply:
I think it makes sense to dedupe and no breaking changes so looks good